src/_includes/layouts/post.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <title>{% if title %} {{ title }} {% else %} {{ site.title }} {% endif %}</title>
+        <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+        <meta content="width=device-width, initial-scale=1.0" name="viewport">
+        <link type="image/x-icon" href="{{ '/' | url }}assets/images/favicon.ico" rel="shortcut icon"/>
+        <!-- Foundation flex grid -->
+        {% include partials/foundationflexgrid.html %}
+        <script type="text/javascript" src="{{ '/' | url }}lib/infusion/infusion-all.js"></script>
+        <script type="text/javascript" src="{{ '/' | url }}assets/js/floe.js"></script>
+        <script type="text/javascript" src="{{ '/' | url }}assets/js/analytics.js"></script>
+        
+    </head>
+    <body>
+        <script type="text/javascript">
+            $(document).ready(function () {
+                floe.setupUIO("../");
+            });
+        </script>
+        {% include partials/uio.html %}
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+        <div class="floe">
+            {% include partials/header.html %}
+            {{content}}
```

Comment:
The following should be added to the template so that it does not have to be duplicated for every news article.

```
<article class="floe-content floe-news-item">
   <h2> News </h2>
       <h3>{{ post.data.title }} </h3>
                <time class="floe-date" datetime="2012-03-06">{{ post.data.date }}</time>
{{content}}
</article>
```

Reply:
Fixed